pkg/k8s/util.go

```diff
@@ -138,31 +138,9 @@ func AsOwner(cc *api.CassandraCluster) metav1.OwnerReference {
 	}
 }
 
-const (
-	// CassandraImage defines the default cassandra image
-	// can be surcharded by the CRD
-	CassandraImage = "sebmoule/cassandra-k8s"
```

Comment:
So happy to see sebmoule go awaaaaaayyyyyy

Reply:
;)